pages/careers-employment/index.md

```diff
@@ -210,9 +210,10 @@ We can support you in all stages of your job search—from returning to work wit
 
 <div class="usa-alert usa-alert-warning">
   <div class="usa-alert-body">
-    <h4 class="usa-alert-heading">How do I get help if I'm homeless or at risk of becoming homeless?<br><a id="crisis-expander-link" onClick="recordEvent({event: 'nav-crisis-homelessness-expander'});">We may be able to help</a>.</h4>
-    <div id="crisis-expander-content" class="expander-content expander-content-closed">
-      <div class="expander-content-inner usa-alert-text">
+    <h4 class="usa-alert-heading">How do I get help if I'm homeless or at risk of becoming homeless?</h4>
+    <div class="form-expanding-group borderless-alert additional-info-container" onClick="recordEvent({event: 'nav-crisis-homelessness-expander'})">
```

Comment:
I think the last thing that we need to do before merging this is get rid of this `onClick` function. I think we can move the GA event into a `data-` property, then reference that in the [`createAdditionalInfoWidget.js`](https://github.com/department-of-veterans-affairs/vets-website/blob/3734cb6c3036a99464c2d2dce105d9ffb539e899/src/applications/static-pages/createAdditionalInfoWidget.js). Something like -

```html
<div class="form-expanding-group borderless-alert additional-info-container" data-event="nav-crisis-homelessness-expander">
...
```

Then, somewhere after [this](https://github.com/department-of-veterans-affairs/vets-website/blob/3734cb6c3036a99464c2d2dce105d9ffb539e899/src/applications/static-pages/createAdditionalInfoWidget.js#L36) line, we can read that property and record it into Google Analytics if it exists -

```js
import recordEvent from 'platform/monitoring/record-event';

// ....

const analyticsEvent = button.dataset.event;
if (analyticsEvent) {
  recordEvent(analyticsEvents);
}
```

This wouldn't help us just yet with the Drupal template, because the alerts are dynamic and so we don't know what to pass into `data-event`. But, we could go ahead and these into the Markdown files, and be in a good position to implement for Drupal once we have a field to plug in there.

Reply:
that sounds like a plan to me. I'll add that as a final thing 👍